x/perpetual/spec/README.md

```diff
@@ -1,40 +1,108 @@
-# Perpetual module
+<!--
+order: 1
+-->
 
-Perpetual module is to support perpetual trading on molecule token pools.
-At initial, isolated perpetual is to be supported and cross perpetual will be supported later.
+# Concepts
 
-## Pool
+The `perpetual` module in the Elys Network facilitates perpetual trading, allowing users to open and close positions on various assets. This guide provides an overview of the key concepts and mechanisms involved in the perpetual module.
 
-Perpetual will use amm pool and will have position records. When the trader closes the position or liquidated, pool tokens will be transferred to the perpetual trader for P&L.
+## Key Concepts
 
-## Perpetual limit
+### Perpetual Trading
 
-Based on pool size perpetual should be limited.
+Perpetual trading allows users to trade assets with leverage without the need for expiry dates on their positions. This type of trading is popular in the cryptocurrency market as it offers flexibility and continuous trading opportunities.
```

Comment:
Is it possible to add mechanism/formula for perpetual positions ?

Reply:
good call out, perpetual module mechanism section was added here https://github.com/elys-network/elys/pull/557/commits/9747e96a7043cd8ae832282c5cbec11487b3174c